x-pack/build.gradle

```diff
@@ -16,17 +16,34 @@ buildscript {
   }
 }
 
+configurations {
+  geolite2
+}
+
 dependencies {
   testImplementation project(':logstash-core')
   testImplementation 'org.assertj:assertj-core:3.8.0'
   testImplementation 'junit:junit:4.12'
+
+  geolite2('org.elasticsearch:geolite2-databases:20191119') {
+    transitive = false
+  }
 }
 
 test {
   exclude '/**'
 }
 
+tasks.register("unzip", Copy) {
```

Comment:
add GeoIP database for unit test

Reply:
This `unzip` task is specific to unzip GeoliteDB, could we rename it as `unzipGeolite` so that the name clearly states what it does